esmvaltool/recipes/recipe_Thermodyn_diagtool.yml

```diff
@@ -0,0 +1,245 @@
+#recipe_Thermodyn_diagtool.yml
+
+---
+documentation:
+  description: |
+        Recipe for the computation of various aspects associated with
+        the thermodynamics of the climate system, such as energy and
+        water mass budgets, meridional enthalpy transports, the Lorenz
+        Energy Cycle and the material entropy production.
+  
+  authors:
+        - lemb_va
+        - kold_ni
+
+  references:
+        - lembo16climdyn
+        - lucarini14revgeop
+
+  projects:
+        - esmval
+        - trr181
+
+datasets:
+    #- {dataset: ACCESS1-0, project: CMIP5, exp: piControl, ensemble: r1i1p1, start_year: 436, end_year: 455}
```

Comment:
Why are there so many commented out datasets here?

Reply:
That was needed for my test, but it does not have to be here on the pull request, so I cleaned it